data/styles/default/include/Icons.mapcss

```diff
@@ -484,12 +484,10 @@ node|z17-[attraction=maze],
 node|z17-[attraction=historic],
 node|z17-[attraction=big_wheel],
 node|z17-[attraction=bumper_car],
-node|z17-[historic=ruins],
-area|z17-[historic=ruins],
+node|z16-[historic=ruins],
```

Comment:
Outdoor style is more suitable for active hiking in bright sunlight outside of cities. Ruins and archeological sites are more like tourist POI, suitable for planning in a general map style, also suitable for cities and old town parts of cities.

Using the same earlier visibility in both styles, including the default one, may help in the easier discovery of interesting places to visit when planning a route.